.github/workflows/todo.yml

```diff
@@ -30,9 +29,8 @@ jobs:
     # if: contains('refs/heads/production', github.ref)  
     env:
       HEROKU_API_KEY: ${{secrets.HEROKU_API_KEY}}
-      TOKEN: ${{ secrets.trelloToken}}
-      KEY: ${{ secrets.trelloKey}}
-      SECRET_KEY: ${{ secrets.trelloSecretKey}}
+      MONGO_DB_CONNECTION: ${{ secrets.MONGO_DB_CONNECTION}}
+      DATABASE_NAME: ${{ secrets.DATABASE_NAME}}
```

Comment:
These env vars aren't actually used by this job. The Test job needs them to run the e2e tests, but you don't need these to deploy to Heroku.

You **could** use the Heroku CLI to set your app's env vars on Heroku, but you aren't currently. I assume you set them manually via the website, in which case you can delete these two lines.